assets/erc20.go

```diff
@@ -16,25 +16,25 @@ type Erc20Params struct {
 
 const ERC20Template = `pragma solidity ^0.5.2;
 
-{{if .Pausable}} import "./lib/oz/contracts/token/ERC20/ERC20Pausable.sol"; {{end}}
-{{if .Cap}}      import "./lib/oz/contracts/token/ERC20/ERC20Capped.sol";   {{end}}
-{{if .Burnable}} import "./lib/oz/contracts/token/ERC20/ERC20Burnable.sol"; {{end}}
-{{if .Mintable}} import "./lib/oz/contracts/token/ERC20/ERC20Mintable.sol"; {{end}}
-				 import "./lib/oz/contracts/token/ERC20/ERC20Detailed.sol";
+{{if .Pausable }}import "./lib/oz/contracts/token/ERC20/ERC20Pausable.sol"; {{end}}
+{{if .Cap	   }}import "./lib/oz/contracts/token/ERC20/ERC20Capped.sol";   {{end}}
+{{if .Burnable }}import "./lib/oz/contracts/token/ERC20/ERC20Burnable.sol"; {{end}}
+{{if .Mintable }}import "./lib/oz/contracts/token/ERC20/ERC20Mintable.sol"; {{end}}
+import "./lib/oz/contracts/token/ERC20/ERC20Detailed.sol";
 
 contract {{.Symbol}}Token is  
-			{{if .Cap}} 	 					ERC20Capped,   {{end}}
-			{{if .Pausable}} 					ERC20Pausable, {{end}}
-			{{if .Burnable}} 					ERC20Burnable, {{end}}
+			{{if .Cap						 }} ERC20Capped,   {{end}}
+			{{if .Pausable					 }} ERC20Pausable, {{end}}
+			{{if .Burnable					 }} ERC20Burnable, {{end}}
```

Comment:
Aren't these equivalent?
```suggestion
			{{if .Burnable}} ERC20Burnable, {{end}}
```

Reply:
they are, it's only for improving readability